apps/core/app/dashboard/products/create/_components/version/version.tsx

```
@@ -0,0 +1,26 @@
+                title: "version", 
+                description: "A freelance graphic and UI/UX designer. I specialise in Web Design , logo and brand development, motion graphics, and offer design services to business of all sizes ....", 
+                version: "1.2.3",
+                checked: true
+            },
+            { 
+                title: "vesrion", 
+                description: "A freelance graphic and UI/UX designer. I specialise in Web Design , logo and brand development, motion graphics, and offer design services to business of all sizes ....", 
+                version: "2.2.4",
+                checked: false
+            }
+        ].map(data => <VersionInfo {...data} />)}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)